### PR TITLE
chore(price-feeder): update prometheus example config

### PR DESCRIPTION
**Pull request by adamewozniak**
Updates the example price-feeder config to properly turn on prometheus

Ref: https://pkg.go.dev/github.com/cosmos/cosmos-sdk/telemetry#Config

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: price-feeder/price-feeder.example.toml [hunk 1]
```diff
@@ -76,9 +76,9 @@ enable-hostname = true
 enable-hostname-label = true
 enable-service-label = true
 enabled = true
-global-labels = [["chain-id", "umee-local-testnet"]]
+global-labels = [["chain_id", "umee-local-testnet"]]
 service-name = "price-feeder"
-type = "prometheus"
+prometheus-retention-time = 100
```
Comment (adamewozniak) [inline price-feeder/price-feeder.example.toml:81]: Thought about adding a reference above the `telemetry` block to [the docs](https://pkg.go.dev/github.com/cosmos/cosmos-sdk/telemetry#Config), but we should comment the whole file with docs if we do that

File: price-feeder/price-feeder.example.toml [hunk 2]
```diff
@@ -76,9 +76,9 @@ enable-hostname = true
 enable-hostname-label = true
 enable-service-label = true
 enabled = true
-global-labels = [["chain-id", "umee-local-testnet"]]
+global-labels = [["chain_id", "umee-local-testnet"]]
```
Comment (zarazan) [inline price-feeder/price-feeder.example.toml:79]: The keys names should be underscores (_) instead of dashes (-) right?

Comment (adamewozniak) [reply price-feeder/price-feeder.example.toml:79]: For this specifically we're using dashes (-), as the cosmos telemetry package we recently switched to uses dashes

File: price-feeder/price-feeder.example.toml (hunk 1 above)
Comment (RafilxTenfen) [reply price-feeder/price-feeder.example.toml:81]: But would be a good idea to comment the whole file '-'

**Review by RafilxTenfen — APPROVED** [review]

**Pull request merged**